.github/workflows/SQL2019.yaml

```diff
@@ -138,13 +141,16 @@ jobs:
           extra-packages:
             #Retrieves most recent odbc pkg from cran to avoid errors seen in older versions.
             #Current version needs R >=3.6 and is currently failing
-            cran::odbc@
-            cran::xml2@
-            rcmdcheck@
+            cran::odbc
+            cran::xml2
+            rcmdcheck
+        continue-on-error: true
```

Comment:
shouldn't we just skip these tasks altogether right now since they fail? All instances of R tasks and tasks where `continue-on-error: true` exists, shouldn't be run.

Reply:
based o our discussions offline will keep the continute-on-error